src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -2,27 +2,29 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.function.Predicate;
+
 import seedu.address.commons.util.ToStringBuilder;
 import seedu.address.logic.Messages;
 import seedu.address.model.Model;
-import seedu.address.model.person.NameContainsKeywordsPredicate;
```

Comment:
side note I think predicates can have their own directory for organisation

Reply:
noted, will do in next iteration